### ISSUE TITLE
Error runing last build (alpha 11)

### ISSUE THREAD

**Issue by edbras** [aam229/local-dependencies#2]
When running the command configure-dependencies, I do get the following errors:
I am using npm version 5.0.3

BigBox:checklist ed$ npm run configure-dependencies

> reis_checklist@0.1.0 configure-dependencies /Users/ed/Develop/Projecten/CodeRepo/Bydehand/bdh-checklist/bdh-checklist-front/checklist
> configure-local-dependencies

? Enter the path to your project (checklist) (node:89756) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): TypeError: Cannot read property 'then' of undefined
(node:89756) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
? Enter the path to your project (checklist) /Users/ed/Develop/Projecten/CodeRepo/Bydehand/bdh-checklist/bdh-checklist-front/checklist

readline.js:982
            throw err;
            ^

TypeError: cb is not a function
    at module.exports (/Users/ed/Develop/Projecten/CodeRepo/Bydehand/bdh-checklist/bdh-checklist-front/checklist/node_modules/run-async/index.js:23:5)
    at PathPrompt.Prompt.filter (/Users/ed/Develop/Projecten/CodeRepo/Bydehand/bdh-checklist/bdh-checklist-front/checklist/node_modules/inquirer-path/node_modules/inquirer/lib/prompts/base.js:128:3)
    at PathPrompt.<anonymous> (/Users/ed/Develop/Projecten/CodeRepo/Bydehand/bdh-checklist/bdh-checklist-front/checklist/node_modules/inquirer-path/node_modules/inquirer/lib/prompts/base.js:56:10)
    at PathPrompt.onFinish (/Users/ed/Develop/Projecten/CodeRepo/Bydehand/bdh-checklist/bdh-checklist-front/checklist/node_modules/inquirer-path/lib/PathPrompt.js:328:12)
    at /Users/ed/Develop/Projecten/CodeRepo/Bydehand/bdh-checklist/bdh-checklist-front/checklist/node_modules/inquirer-path/lib/PathPrompt.js:314:18
    at module.exports (/Users/ed/Develop/Projecten/CodeRepo/Bydehand/bdh-checklist/bdh-checklist-front/checklist/node_modules/run-async/index.js:23:5)
    at PathPrompt.Prompt.filter (/Users/ed/Develop/Projecten/CodeRepo/Bydehand/bdh-checklist/bdh-checklist-front/checklist/node_modules/inquirer-path/node_modules/inquirer/lib/prompts/base.js:128:3)
    at PathPrompt.onSuccess (/Users/ed/Develop/Projecten/CodeRepo/Bydehand/bdh-checklist/bdh-checklist-front/checklist/node_modules/inquirer-path/lib/PathPrompt.js:290:12)
    at /Users/ed/Develop/Projecten/CodeRepo/Bydehand/bdh-checklist/bdh-checklist-front/checklist/node_modules/inquirer-path/lib/PathPrompt.js:222:18
    at module.exports (/Users/ed/Develop/Projecten/CodeRepo/Bydehand/bdh-checklist/bdh-checklist-front/checklist/node_modules/run-async/index.js:23:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! reis_checklist@0.1.0 configure-dependencies: `configure-local-dependencies`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the reis_checklist@0.1.0 configure-dependencies script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ed/.npm/_logs/2017-06-15T15_40_08_176Z-debug.log
B

**Comment by aam229** [comment]
Yeah I noticed this issue when I tried to run the configure-dependencies script as well. It's due to incompatible versions of inquirer used in the `local-dependencies` module and the `inquirer-path` module. I'm working on an update to `inquirer-path` but it's unfortunately not as simple as updating the `inquirer` dependency. I'll update you when it's ready, which hopefully will be sometime this week-end.
